css/README.md

```diff
@@ -410,3 +413,47 @@ When using HSL or RGB notation, always add a single space after a comma (,) and
 We prefer using mixins over functions. If a function is absolutely necessary, document where it is used and why it wasn't possible to create a mixin.
 
 **[⬆ back to top](#table-of-contents)**
+
+## CSS Modules
+
+[CSS Modules documentation](https://github.com/css-modules/css-modules)
+
+- Prefer using CSS Modules over global css code
+- Use camelCase for class names
+- Always use `.module.css` or `.module.scss` extension
+- Do not expect any scss variables or mixins to be defined. You have to import them yourself.
+- Prefer Sass variables over CSS Module variables
```

Comment:
I don't think we can or want to get rid of sass completely. Therefore I think it's the best for now to not mix css-module variables and sass variables.

Reply:
yes prefer sass variables since they just work and sass in general is more expressive

Reply:
Wouldn't forbid the 'export' feature. Could be useful someday.